src/loaders/core_text.rs

```diff
@@ -779,6 +801,11 @@ fn format_to_cg_color_space_and_image_format(format: Format) -> Option<(CGColorS
     }
 }
 
+#[link(name = "CoreGraphics", kind = "framework")]
+extern {
```

Comment:
Looks like upstream has this method now; can we switch to using it?

Reply:
@pcwalton it's been merged, but it looks like `core-text` hasn't cut a new release yet. Is that something you could help out with?

Reply:
I'll publish a new version when https://github.com/servo/core-foundation-rs/pull/294 is merged.